bluepill/src/BPRunner.m

```diff
@@ -213,24 +213,35 @@ - (int)runWithBPXCTestFiles:(NSArray<BPXCTestFile *> *)xcTestFiles {
         [BPUtils printInfo:ERROR withString:@"Packing failed: %@", [error localizedDescription]];
         return 1;
     }
-    if (bundles.count < numSims) {
-        [BPUtils printInfo:WARNING
-                withString:@"Lowering number of parallel simulators from %lu to %lu because there aren't enough test bundles.",
-                            numSims, bundles.count];
-        numSims = bundles.count;
-    }
     if (self.config.cloneSimulator) {
         self.testHostForSimUDID = [bpSimulator createSimulatorAndInstallAppWithBundles:xcTestFiles];
         if ([self.testHostForSimUDID count] == 0) {
             return 1;
         }
     }
+
+    int repeatTestsCount = [self.config.repeatTestsCount intValue];
```

Comment:
Won't this PR add repeat test count 2x?

Reply:
Oh, you're right. I think it was a merge conflict. Updating.